src/styles/global.css

```
@@ -1,9 +1,537 @@
-	@apply mx-auto p-6 lg:px-32 max-w-5xl;
+:root {
+	--paper: oklch(0.16 0.012 70);
+	--paper-2: oklch(0.21 0.014 70);
+	--ink: oklch(0.96 0.008 80);
+	--ink-2: oklch(0.78 0.010 80);
+	--muted: oklch(0.62 0.010 80);
+	--rule: oklch(0.30 0.014 70);
+	--accent: oklch(0.58 0.14 38);
+	--accent-soft: oklch(0.30 0.06 38);
+
+	--font-display: "Newsreader", ui-serif, Georgia, serif;
```

Stylelint (17.14.1)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)